fabtests/common/shared.c

```diff
@@ -2786,11 +2785,9 @@ int ft_sendmsg(struct fid_ep *ep, fi_addr_t fi_addr,
 		tagged_msg.tag = ft_tag ? ft_tag : tx_seq;
 		tagged_msg.ignore = 0;
 
-		ret = fi_tsendmsg(ep, &tagged_msg, flags);
-		if (ret) {
-			FT_PRINTERR("fi_tsendmsg", ret);
-			return ret;
-		}
+		FT_POST(fi_tsendmsg, ft_progress, txcq, tx_seq,
+			&tx_cq_cntr, "tsendmsg", ep, &tagged_msg,
+			flags);
```

Comment:
The old code didn't increase tx_seq. Is that considered a bug being fixed by this patch? In addition, should similar changes be made to `ft_recvmsg`?

Reply:
> The old code didn't increase tx_seq. Is that considered a bug being fixed by this patch?

I think so. Currently this function is only used by inject.c test, I didn't see it caused an issue though.


> should similar changes be made to ft_recvmsg?

Yes we should, but it seems this function is obsolete.